### PR TITLE
meta-psa: add README

### PR DESCRIPTION
**Pull request by jonathanhaigh-arm**
For IOTMBL-2150: Write meta-psa/mbed-crypto recipe documentation

### CODE REVIEW

File: meta-psa/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# meta-psa OpenEmbedded layer
+
+The meta-psa OpenEmbedded layer provides reference implementations and tests of
```
Comment (iriark01) [inline meta-psa/README.md]:
```suggestion
The meta-psa OpenEmbedded layer provides reference implementations and tests for
```

File: meta-psa/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## PSA
+The Arm Platform Security Architecture (PSA) is a holistic set of threat
+models, security analyses, hardware and firmware architecture specifications,
+and an open source firmware reference implementation. PSA provides a recipe,
+based on industry best practice, that allows security to be consistently
+designed in, at both a hardware and firmware level.
+
+For more information, see the [PSA website][psa].
+
+## Recipes
```
Comment (iriark01) [inline meta-psa/README.md]: If there's just the one, please don't use a bulleted list.

File: meta-psa/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# meta-psa OpenEmbedded layer
+
+The meta-psa OpenEmbedded layer provides reference implementations and tests of
+the Arm Platform Security Architecture APIs.
+
+## PSA
+The Arm Platform Security Architecture (PSA) is a holistic set of threat
```
Comment (iriark01) [inline meta-psa/README.md]: Is that copied from the PSA site? It's a little marketingy.

File: meta-psa/README.md (hunk 2 above)
Comment (jonathanhaigh-arm) [reply meta-psa/README.md]: There will be more very soon. https://github.com/ARMmbed/meta-mbl/pull/598 adds a new recipe (psa-arch-tests) in this layer.

I suppose maybe I should just list psa-arch-tests in the list and wait until https://github.com/ARMmbed/meta-mbl/pull/598 is merged before merging this one.

File: meta-psa/README.md (hunk 3 above)
Comment (jonathanhaigh-arm) [reply meta-psa/README.md]: It does appear on the PSA site. It also appears at the start of each of the actual PSA spec documents and as an introductory statement at https://github.com/ARMmbed/mbed-crypto and https://github.com/ARM-software/psa-arch-tests. I'm open to different wording, but I don't feel qualified to summarize what PSA is myself.

Comment (iriark01) [reply meta-psa/README.md]: If that's how they phrased it - keep it

File: meta-psa/README.md (hunk 2 above)
Comment (jonathanhaigh-arm) [reply meta-psa/README.md]: Added reference to psa-arch-tests recipe in https://github.com/ARMmbed/meta-mbl/pull/600/commits/bba62cdb08b673c41084bb26ea2fcfec50e4f8d7.

**Comment by jonathanhaigh-arm** [comment]
@iriark01, can you re-review after the changes please?

**Comment by jonathanhaigh-arm** [comment]
Thanks @iriark01.

@jjohnson-arm, when I hover my mouse pointer over the tick icon for Irit's approaval I'm told that "only reviews by reviewers with write access count toward mergability". Can you approve too please?

**Pull request merged**